build/gen_stub.php

```diff
@@ -1014,6 +1031,10 @@ public function getMethodSynopsisFilename(): string {
         return $this->getDeclarationClassName() . "_{$this->methodName}";
     }
 
+    public function getAttributeName(): string {
+        return $this->methodName;
```

Comment:
The method name needs to be lowercased. Example: `ZipArchive::setEncryptionIndex()`